exec/tracking_summary_perfect_tracks.R

```diff
@@ -25,42 +25,42 @@ option_list <- list(
 
 opt <- parse_args(OptionParser(option_list = option_list))
 
-foo <- capture.output(synLogin())
-
-
-## ----get-tracking-results, message = FALSE---------------------------------
 if (stringr::str_detect(opt$tracking_file, "^syn.*")) {
-  trackingResults <- neurolincsscoring::syn_get_tracking_submission_file(opt$tracking_file)
+  syn_tracking_file <- TRUE
+  get_tracking_file_fxn <- neurolincsscoring::syn_get_tracking_submission_file
```

Comment:
I'm concerned this won't work if we do pass a Synapse ID to the `--tracking_file` flag. It looks like we are calling `syn_get_tracking_submission_file` if the argument is a Synapse ID -- which then immediately tries to `synGet` despite the fact that we haven't logged in yet.

Reply:
ohh, I see now. You aren't actually calling the function yet.

Reply:
anonymous function, FTW!